src/pages/guides/javascript/how-to-create-an-spl-token-on-solana.md

```diff
@@ -0,0 +1,462 @@
+---
+title: How to Create an Spl Token On Solana
+metaTitle: How to Create an Spl Token On Solana
+description: Learn how to create an Spl Token on the Solana blockchain with Metaplex packages.
+# remember to update dates also in /components/guides/index.js
+created: '06-16-2024'
+updated: '06-21-2024'
+---
+
+This guide walks you through setting up and minting your very own token on the Solana blockchain using the Metaplex Umi client wrapper and the Mpl Toolbox package with Javascript.
```

Comment:
wdyt about "minting your very own SPL Token".  Also note, the first time we say SPL Token should we call out that SPL stands for Solana Program Library?